src/WorkflowGui/Resources/config/pimcore/routing.yml

```diff
@@ -20,6 +20,16 @@ workflow_gui.admin.delete:
     methods: [POST]
     defaults: { _controller: WorkflowGuiBundle:Workflow:delete }
 
+workflow_gui.admin.visualize:
+    path: /admin/workflow/visualize
+    methods: [GET]
+    defaults: { _controller: WorkflowGuiBundle:Workflow:visualize }
+
+workflow_gui.admin.visualizeImage:
```

Comment:
use snake case pls, also in path

Reply:
You meant visualizeImage -> visualize_image? Have changed that.
I wonder because you listed the lines of the "visualize" route where is nothing I can think of being convertable to snake case.

Reply:
the name of the route, and the path of the route is what I meant

Reply:
thats what you changed anyway, so that is fine now.